File: Code Generation/responses/cpp/claude/easy/find-the-largest-almost-missing-integer.cpp
#include <vector>
#include <unordered_map>
#include <algorithm>
using namespace std;

class Solution {
public:
    int largestInteger(vector<int> nums, int k) {
        int n = nums.size();
        unordered_map<int, int> count;
        
        for (int i = 0; i <= n - k; i++) {
            unordered_map<int, bool> seen;
            for (int j = i; j < i + k; j++) {
                if (!seen[nums[j]]) {
                    count[nums[j]]++;
                    seen[nums[j]] = true;
                }
            }
        }
        
        int ans = -1;
        for (auto& [val, cnt] : count) {
            if (cnt == 1) ans = max(ans, val);
        }
        return ans;
    }
};

File: Code Generation/responses/python/claude/easy/find-the-largest-almost-missing-integer.py
from typing import List
from collections import defaultdict

class Solution:
    def largestInteger(self, nums: List[int], k: int) -> int:
        count = defaultdict(set)
        for i in range(len(nums) - k + 1):
            for x in set(nums[i:i+k]):
                count[x].add(i)
        return max((x for x, s in count.items() if len(s) == 1), default=-1)